GetWowpediaInfo.lua

```diff
@@ -208,24 +323,84 @@ end
 
 table.sort(apiKeys)
 
+out.write(out, "--- @class unknown @ unknown type\n\n")
+
 for _,k in pairs(apiKeys) do
+    local preFunction = ""
     local functionName = k
-    local returnValues
-    local argValues
+    local argValues = ""
+
+    if api_entries[k].description then
+        preFunction = preFunction .. "--- ### " .. api_entries[k].description .. "\n"
```

Comment:
I'm not sure about using this H3 Markdown tag here. I'm guessing you wanted the text to appear bigger, but I think as it is just the description of the function it should not use a header tag. The documentation for the C_ apis are not using this header tag.